cfgov/locale/es/LC_MESSAGES/django.po

```diff
@@ -1,14 +1,196 @@
+"Language: es\n"
+
+#: v1/_includes/molecules/global-search.html
+msgid "Search"
+msgstr "Buscar"
+
+msgid "Close"
+msgstr "Cerrar"
+
+msgid "Search the CFPB"
+msgstr "Buscar la CFPB"
+
+msgid "Submit a Complaint"
+msgstr "Presentar una queja"
+
+msgid "Mortgage"
+msgstr "Hipoteca"
+
+#: v1/ask-cfpb/category-page.html
+msgid "results"
+msgstr "resultados"
+
+msgid " results"
+msgstr " resultados"
+
+msgid "Read answer"
+msgstr "Leer la respuesta"
+
+msgid "/ask-cfpb/"
+msgstr "/es/obtener-respuestas/"
+
+msgid "Previous"
+msgstr "Anterior"
+
+msgid "Next"
+msgstr "Siguiente"
+
+msgid "No results found for"
+msgstr "No se encontraron resultados para"
+
+msgid "Please enter a search term in the box above."
+msgstr "Por favor ingrese un término de búsqueda en el cuadro de arriba."
+
+#: cfgov/jinja2/v1/ask-cfpb/answer-page.html
+msgid "updated"
+msgstr "actualizado"
+
+msgid "Please do not share any personally identifiable information (PII), including, but not limited to: your name, address, phone number, email address, Social Security number, account information, or any other information of a sensitive nature."
+msgstr "No incluya información confidencial, como su nombre, información de contacto, número de cuenta o número de seguro social en este campo."
+
+msgid "Read full answer"
+msgstr "Leer la respuesta completa"
+
+msgid "Hide full answer"
+msgstr "Esconder la respuesta completa"
+
+msgid "Explore related resources"
+msgstr "Explora recursos relacionados"
+
+msgid "Related questions"
+msgstr "Preguntas similares"
+
+#: cfgov/jinja2/v1/ask-cfpb/answer-search-results.html
+msgid "Can't find your question?"
+msgstr "¿No encuentra su pregunta?"
+
+msgid "Ask CFPB"
+msgstr "Obtener respuestas"
+
+msgid "Showing results for "
+msgstr "Mostrando resultados para "
+
+msgid "Showing"
+msgstr "Mostrando"
+
+msgid "of"
+msgstr "de"
+
+msgid "results for"
+msgstr "resultados para"
+
+msgid "Search instead for"
+msgstr "Buscar en lugar para"
+
+#: cfgov/jinja2/v1/ask-cfpb/landing-page.html
+msgid "Browse questions by category"
+msgstr "Examinar las preguntas por categoría"
+
+#: cfgov/jinja2/v1/_includes/blocks/feedback.html
+msgid "Was this answer helpful to you?"
+msgstr "¿Te resultó útil esta respuesta?"
+
+msgid "Additional comment"
+msgstr "Comentario adicional"
+
+msgid "Yes"
+msgstr "Sí"
+
+msgid "optional"
+msgstr "opcional"
+
+#: v1/_includes/ask/ask-search-bar.html
+msgid "/ask-cfpb/search"
+msgstr "/es/obtener-respuestas/buscar"
+
+msgid "Don't see what you're looking for? <span class=\"break\">Search for your question.</span>"
+msgstr "¿No ve lo que está buscando? <span class=\"break\">Busca su pregunta.</span>"
+
+#: cfgov/jinja2/v1/_includes/atoms/tagline.html
+msgid "An official website of the United States government"
+msgstr "Un sitio web oficial del gobierno federal de los Estados Unidos"
+
+#: cfgov/jinja2/v1/_includes/molecules/pagination.html
+msgid "Page"
+msgstr "Página"
+
+msgid " of "
```

Comment:
Do we want leading and trailing spaces in these translations? It seems like these should be stripped and added in the context of where ` of ` appears.

Reply:
Removed in f06ddeeb84ff023e7b717dc3507e5ec1392e72c0

Reply:
I'd be careful about removing spaces. They're usually intentional because of where they are being used, and removing a space where it's needed will break the translation. If the translation is working, you should not strip.

Reply:
@higs4281 Also a good point. Do you have specific concerns about these updates? Handling the spacing in the templates works for me locally, but I can revert if you think the changes could be problematic.

Reply:
If the translation works, that's great, and it's clearly better not to have leading or trailing spaces.

I was only worried about cases where such spaces might be necessary because of some complex translation, and I'd hate to disable someone's work.